Makefile

```diff
@@ -104,7 +104,7 @@ licenses: .state/docker-build-base
 deps: .state/docker-build-base
 	docker compose run --rm base bin/deps
 
-translations: .state/docker-build-base
+translations: .state/docker-build-base .state/docker-build-static
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** now that the build is part of webpack, is this dependency necessary? From [the inline doc](https://github.com/pypi/warehouse/pull/15612/files?w=1#diff-55479889313695828cc288bac91197837669dcc5d650b665c09bdb8b4dce211cR20) it seems like `make static` should depend on translations instead?‏

Reply:
Agreed, I don't think this is necessary.